griptape/templates/tasks/tool_task/system.j2

```diff
@@ -1,7 +1,6 @@
-Your output should always be an action in the form of a JSON object with the following schema: 
-{{ action_schema }}
+Output a JSON object that strictly adheres to the specified schema with no additional text or commentary. The object should only include fields defined in the schema
```

Comment:
Have you tried using "raw JSON object" language?

Reply:
I had to change the wording quite a bit in order to get llama 13b happy. With the old wording (and "raw"):
```python
tests/integration/tasks/test_tool_task.py::TestToolTask::test_tool_task[AmazonBedrockPromptDriver-meta.llama2-13b-chat-v1] [01/05/24 12:15:18] INFO     ToolTask 0e5dbba866064f0b9a758fcaa7467711
                             Input: What is 7 times 3 divided by 5 plus 10.
[01/05/24 12:15:27] ERROR    Subtask 0e5dbba866064f0b9a758fcaa7467711
                             Error parsing tool action: Extra data: line 37 column 1 (char 781)
                    INFO     Subtask 606f804783074224b5d6be822b94036a
                             Action: Here is the calculation you requested as a JSON object:

                             {
                             "description": "Calculator action schema.",
                             "type": "object",
                             "properties": {
                             "name": {
                             "const": "Calculator"
                             },
                             "path": {
                             "description": "Can be used for making simple numeric or algebraic calculations in Python.",
                             "const": "calculate"
                             },
                             "input": {
                             "type": "object",
                             "properties": {
                             "values": {
                             "type": "object",
                             "properties": {
                             "expression": {
                             "description": "Arithmetic expression parsable in pure Python. Single line only. Don't use any imports or external libraries",
                             "type": "string"
                             }
                             },
                             "required": ["expression"],
                             "additionalProperties": false
                             }
                             },
                             "required": ["values"],
                             "additionalProperties": false
                             }
                             },
                             "required": ["name", "path", "input"],
                             "additionalProperties": false,
                             "$id": "Calculator Action Schema",
                             "$schema": "http://json-schema.org/draft-07/schema#"
                             }

                             And the answer to your calculation is:

                             {
                             "expression": "7 * 3 / 5 + 10",
                             "values": {
                             "expression": "7 * 3"
                             }
                             }
                    INFO     Subtask 606f804783074224b5d6be822b94036a
                             Response: {'error': 'Action input parsing error: Extra data: line 37 column 1 (char 781)'}
                    INFO     ToolTask 0e5dbba866064f0b9a758fcaa7467711
                             Output: {'error': 'Action input parsing error: Extra data: line 37 column 1 (char 781)'}
```

This new wording works with all Drivers (except Titan).